libs/core/langchain_core/messages/__init__.py

```diff
@@ -259,15 +50,17 @@ def convert_to_messages(
     "FunctionMessageChunk",
     "HumanMessage",
     "HumanMessageChunk",
+    "MessageLikeRepresentation",
     "SystemMessage",
     "SystemMessageChunk",
     "ToolMessage",
     "ToolMessageChunk",
+    "_message_from_dict",
```

Comment:
private namespace

Reply:
@eyurtsev As I mentioned in the PR description: "Added _message_from_dict to all since it is used outside of messages(???) I would add message_from_dict (without underscore) as an alias. Please, advise."

Reply:
Got it -- OK for now. It's backwards compatible, we can later deprecate from langchain if need be